anvil/distributions.py

```diff
@@ -157,7 +160,7 @@ def log_density(self, sample):
     def pdf(self):
         x = torch.linspace(-self.radius, self.radius, 10000)
         dens = 2 / (pi * self.radius ** 2) * torch.sqrt(self.radius ** 2 - x ** 2)
-        return x + self.mean, dens
+        return ((x + self.mean, dens),)
```

Comment:
seems weird that we need to return a tuple of length one, where the first element is a tuple

Reply:
Yeah that's all it is. I'm trading weirdness here for neatness in the function that plots them, but perhaps it makes more sense to have `pdf` return something sensible.